upgrade_tests/test_existance_relations/cli/test_contenthosts.py

```diff
@@ -46,8 +43,6 @@ def test_positive_contenthosts_by_name(pre, post):
     assert existence(pre, post)
 
 
-@dont_run_to_upgrade(['6.8', '6.9', '6.10', '6.11'])
-@pytest_skip_if_bug_open('bugzilla', 1461397)
```

Comment:
Though WONTFIX we can keep the test
With 6.3 behavior had changed (applicable errata: `0` -> `""`) and then stayed the same